src/analyses/apron/linearTwoVarEqualityAnalysis.apron.ml

```diff
@@ -0,0 +1,36 @@
+(** {{!RelationAnalysis} Relational integer value analysis} using an OCaml implementation of the linear two-variable equalities domain ([...]).
+
+    @see <http://doi.acm.org/10.1145/2049706.2049710> A. Flexeder, M. Petter, and H. Seidl Fast Interprocedural Linear Two-Variable Equalities. *)
+
+open Analyses
+include RelationAnalysis
+
+let spec_module: (module MCPSpec) Lazy.t =
+  lazy (
+    let module AD = LinearTwoVarEqualityDomain.D2  in
+    let module RD: RelationDomain.RD =
+    struct
+      module V = LinearTwoVarEqualityDomain.V
+      include AD
+    end
+    in
+    let module Priv = (val RelationPriv.get_priv ()) in
+    let module Spec =
+    struct
+      include SpecFunctor (Priv) (RD) (RelationPrecCompareUtil.DummyUtil)
+      let name () = "lin2vareq"
+    end
+    in
+    (module Spec)
+  )
+
+let get_spec (): (module MCPSpec) =
+  Lazy.force spec_module
+
+let after_config () =
+  let module Spec = (val get_spec ()) in
+  MCP.register_analysis (module Spec : MCPSpec);
+  GobConfig.set_string "ana.path_sens[+]"  (Spec.name ())
```

Comment:
Why does this analysis declare itself to be path-sensitive?

Reply:
There's some discussion on this at the bottom here: https://github.com/goblint/analyzer/pull/291

Reply:
Ok it seems one of our privatizations (`module ProtectionBasedPriv (Param: ProtectionBasedPrivParam): S = functor (RD: RelationDomain.RD) ->`) uses it sometimes, and the representative is defined  based on only that, so it's fine.